test/libsolidity/semanticTests/errors/weird_name.sol

```diff
@@ -1,10 +0,0 @@
-error error(uint a);
-contract C {
```

Comment:
Could you keep this case as a syntax test?

Reply:
I'll add this, and push now.